core/sql/regress/executor/TEST002

```diff
@@ -661,7 +661,9 @@ left outer join (
   ) T) T) T) T) T) T
 on (t002t6.c = T.a)
 ;
+
 control query shape merge_join(cut,cut);
+cqd PCODE_OPT_LEVEL '0';
```

Comment:
This is worrisome actually. Is there a follow-on JIRA to fix the Pcode problems?